content/how-to/cloud-files/getting-started-with-cloud-files-and-cdn/index.md

```diff
@@ -1,49 +1,63 @@
 ---
 permalink: getting-started-with-cloud-files-and-cdn/
-audit_date:
+audit_date: '2021-03-30'
 title: Getting started with Cloud Files and CDN
 type: article
 created_date: '2016-01-14'
 created_by: Stephanie Fillmon
-last_modified_date: '2018-10-23'
-last_modified_by: Kate Dougherty
+last_modified_date: '2021-03-30'
+last_modified_by: Carlos Arriaga
 product: Cloud Files
 product_url: cloud-files
 ---
 
-This article provides an overview of Rackspace Cloud Files and its
+This article provides an overview of Rackspace Technology's Cloud Files and its
```

Comment:
I don't think you should make Rackspace Technology possessive here

Reply:
Let's go with Rackspace Technology Clod Files instead. How's that @robb-romans ?

Reply:
agreed.  Style Guide says to avoid company possessive.  Say "The Microsoft system" rather than "Microsoft's system".  I missed that!!